x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_actions/rule_actions_pli_essentials.cy.ts

```diff
@@ -31,7 +31,7 @@ const rule = getNewRule();
 describe(
   'Rule actions PLI essentials product tier',
   {
-    tags: ['@serverless'],
+    tags: ['@serverless', '@brokenInServerless'],
```

Comment:
If we're skipping this suite, should there be a corresponding issue to account for fixing that?

Reply:
This is outdated now. The broken in serverless tag is removed as it is working as of yesterday!